scripts/install-lint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+GOBIN="$(go env GOPATH)/bin"
```

Comment:
I was hoping github would show the diff even though the file was renamed.  The command above works equally well for defaulted and custom GOPATH valus.